app/controllers/application_controller.rb

```diff
@@ -134,9 +135,24 @@ def authenticate_token
       return false if @session.nil?
       @current_user = User.find(@session.uuid)
       SSOService.extend_session!(@session, @current_user)
+      set_sso_cookie
     end
   end
 
+  def set_sso_cookie
+    contents = [expiryunix, @current_user.loa, @current_user.mvi.icn,
+                @current_user.mhv_correlation_id, @current_user.edipi]
```

Comment:
Per discussions yesterday, looks like we'll only need the expiry, correlation id, and icn. That should be `mvi_icn`, as well.